plugins/native/configs/config.d/10_native.json

```diff
@@ -61,5 +61,12 @@
       "anti-aliasing": false,
       "translucency-support": false
     }
+  },
+  {
+    "match": ".*(mdl)",
+    "options": {
+      "up": "+Z",
+      "animation-index": "-1"
```

Comment:
So most animated models have multiples animations baked in and all animations should be enabled by default ?